### ISSUE TITLE
Working Search bar 

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/spmssite#92]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/gbowne1/spmssite/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/gbowne1/spmssite/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/gbowne1/spmssite/issues) that already report this problem, without success.

### Current Behavior

For Google Search to work inside the search bar..

I don't know what it would take to get it to only search the page/site itself.

```js

    document.getElementById('frmSearch').onsubmit = function() {
        window.location = 'http://www.google.com/search?q=site:yoursitename.com ' + document.getElementById('txtSearch').value;
        return false;
    }
```

### Expected Behavior

Actually working search bar!

### Steps to Reproduce

Try it!

**Comment by gbowne1** [comment]
@ketsebaotgizachew @ketse-py

**Comment by VandanaPabale** [comment]
can I resolve this issue?

**Comment by gbowne1** [comment]
Yes @VandanaPabale

**Comment by gbowne1** [comment]
Closed by PR #129
